src/_includes/layouts/base.njk

```diff
@@ -30,6 +36,52 @@
             </article>
         </main>
         {% include "partials/global/footer.njk" %}
-        {% uioInit %}
+        {# {% uioInit %} #}
+        <script>
+        fluid.defaults("fluid.uiOptions.multilingual", {
+            gradeNames: ["fluid.uiOptions"],
+            distributeOptions: {
+                source: "{that}.options.locale",
+                target: "{that messageLoader}.options.resourceOptions.locale"
+            },
+            listeners: {
+                "onPrefsEditorReady.addLanguageAttributesToBody": {
+                    func: "fluid.uiOptions.multilingual.addLanguageAttributesToBody",
+                    args: ["{that}.prefsEditorLoader.prefsEditor.container", "{that}.options.locale", "{that}.options.direction"]
+                }
+            }
+        });
```

Comment:
Assuming you made the changes from the previous comment to use `defaultLocale`, and also you could remove the `fluid.uiOptions.multilingual.addLanguageAttributesToBody` function.

```suggestion
            listeners: {
                "onPrefsEditorReady.addLanguageAttributesToBody": {
                    this: "{that}.prefsEditorLoader.prefsEditor.container",
                    method: "attr",
                    args: [{
                        "lang": "{that}.options.prefsEditorLoader.defaultLocale",
                        "dir": "{that}.options.prefsEditorLoader.direction"
                    }]
                }
            }
        });
```

You may be able to replace `"{that}.options.prefsEditorLoader"` with `"{prefsEditorLoader}.options"`.